### PR TITLE
[WIP] Scaffold for mark-test

### PR DESCRIPTION
**Pull request by kanitw**

### CODE REVIEW

**Comment by kanitw** [comment]
@ayush29feb  Don't forget that you have this on your plate :)   Once you're ready for review, please assign it back to me.   

We started this branch a long ago, you might want to rebase to the current master :)

**Comment by domoritz** [comment]
@ayush29feb Can you please finish this? I guess you need to rebase this.

**Comment by ayush29feb** [comment]
@domoritz Yeah, I am working on it. Would it make more sense to finish and then rebase or first rebase and then finish.

**Comment by domoritz** [comment]
Definitely rebase first.

File: test/compile/mark.test.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      // TODO
+    });
+
+    it('should sort by an order field if it has an order field', function() {
+      // TODO
+    });
+  });
+
+  describe('Bar', function() {
+    describe('with a color field', function() {
```
Comment (kanitw) [inline test/compile/mark.test.ts]: Actually, this should be aggregate, with a color field. 

(Because if it is not aggregate, we never stack)

**Comment by kanitw** [comment]
FYI I moved a test related to stacked bar from `stack.test.ts` to `mark.test.ts` in the huge composition refactor branch: https://github.com/vega/vega-lite/pull/1192

See https://github.com/vega/vega-lite/blob/unit/test/compile/mark/mark.test.ts

**Comment by ayush29feb** [comment]
@kanitw thanks for letting me know.

**Pull request closed without merging**